### DISCUSSION TITLE
Possible to insert a newline in the prompt?

### DISCUSSION THREAD

**Discussion by CubicPants** [danny-avila/LibreChat#2309] [Help Wanted]
Hi All,

As per subject. Is there a way to enter multiple lines at the prompt? I know it can be done with json and \n and copy  paste etc. But would be nice to just type it straight into the prompt box.

Regards.

**Comment by fuegovic** [comment]
yes, you can do it from the input box with <kbd>shift</kbd>+<kbd>Enter</kbd>

**Reply by CubicPants** [reply]
I tried Alt and Ctrl, didn't think to try Shift!
Thank you fuegovic.
